### PR TITLE
GPII-3158: Update gpii-grunt-mdjson-lint to remove outdated ast-markdown dependency.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [https://issues.gpii.net/browse/GPII-3158](https://issues.gpii.net/browse/GPII-3158) for details.

### CODE REVIEW

**Comment by the-t-in-rtf** [comment]
@amb26, we can review this at any time, but should only merge after [the related gpii-grunt-mdjson-lint](github.com/GPII/gpii-grunt-mdjson-lint/pull/1) pull is merged.

File: src/load-npm-tasks-properly.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"use strict";
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+var resolve = require("fluid-resolve");
+
+fluid.registerNamespace("gpii.grunt.lintAll");
+gpii.grunt.lintAll.fixGruntTaskLoading = function (grunt) {
+    grunt.loadNpmTasksProperly = function (name) {
```
Comment (amb26) [inline src/load-npm-tasks-properly.js]: Whilst the comment for this method needn't be as ad hominem as the original one, we should have some kind of fairly detailed explanation of what the problem is here and why this approach is a solution to it

Comment (the-t-in-rtf) [reply src/load-npm-tasks-properly.js]: OK, it's definitely gonna be detailed, as there are two problems, and our fix only addresses one.  I'll take a stab at it and you can comment on that shortly.

Comment (the-t-in-rtf) [reply src/load-npm-tasks-properly.js]: OK, call me Ishmael, but I tried to keep it as brief as I could. @amb26, take a look and let me know if this is enough for now.

**Comment by the-t-in-rtf** [comment]
@amb26, this pull now includes [my first pass at implementing global excludes](https://issues.gpii.net/browse/GPII-3245).

**Comment by the-t-in-rtf** [comment]
Thanks, @amb26.  I cut the release and updated the forward-facing version.

**Comment by the-t-in-rtf** [comment]
I also invited you to be an admin in the new npm package.

**Pull request merged**